internal/log/log.go

```diff
@@ -83,31 +83,18 @@ func With() zerolog.Context {
 	return Logger().With()
 }
 
-// Level creates a child logger with the minimum accepted level set to level.
-func Level(ctx context.Context, level zerolog.Level) *zerolog.Logger {
-	l := contextLogger(ctx).Level(level)
-	return &l
-}
-
 // Debug starts a new message with debug level.
 //
 // You must call Msg on the returned event in order to send the event.
-func Debug(ctx context.Context) *zerolog.Event {
-	return contextLogger(ctx).Debug()
+func Debug() *zerolog.Event {
+	return log.Debug()
 }
 
 // Info starts a new message with info level.
 //
 // You must call Msg on the returned event in order to send the event.
-func Info(ctx context.Context) *zerolog.Event {
-	return contextLogger(ctx).Info()
-}
-
-// Warn starts a new message with warn level.
-//
-// You must call Msg on the returned event in order to send the event.
-func Warn(ctx context.Context) *zerolog.Event {
```

Comment:
note to self: `Warn()` is removed because we say not to use the "warning" log level in our [Contributing guide](https://www.pomerium.com/docs/community/contributing#logging)